picasso/src/main/java/com/squareup/picasso/RequestCreator.java

```diff
@@ -735,7 +737,17 @@ public void into(ImageView target, Callback callback) {
 
   private Drawable getPlaceholderDrawable() {
     if (placeholderResId != 0) {
-      return picasso.context.getResources().getDrawable(placeholderResId);
+      if (Build.VERSION.SDK_INT >= 21) {
+        return picasso.context.getDrawable(placeholderResId);
+      } else if (Build.VERSION.SDK_INT >= 16) {
+        return picasso.context.getResources().getDrawable(placeholderResId);
+      } else {
+        final int resolvedId;
+        TypedValue sTempValue = new TypedValue();
+        picasso.context.getResources().getValue(placeholderResId, sTempValue, true);
```

Comment:
for my own understanding, what's wrong with `picasso.context.getResources().getDrawable(placeholderResId);` on < 16?

Reply:
Check out the Javadoc of the method. I was also curious as to why this was needed this morning!